ArduCopter/Copter.h

```diff
@@ -211,6 +211,7 @@ class Copter : public AP_HAL::HAL::Callbacks {
     ParametersG2 g2;
 
     // main loop scheduler
+#define MAIN_LOOP_SCHEDULER_HZ 400
```

Comment:
Soooo.... what happens when someone sets `SCHED_LOOP_RATE` to something else?  It's really going to be better to just stash away a `_roi_yaw_last_updated_ms` value here, I think.

Also, I'm developing a pet hate of `#define`s for numeric constants :-)    @WickedShell is promoting having these values directly in the code where they are only used once - so in this case you'd have a `const uint8_t _yaw_update_interval_ms = 10;` in the `roi_yaw` method.

Reply:
I think that it is better to refer to this value as the loop rate value.
libraries\AP_Scheduler\AP_Scheduler.cpp
    // @Param: LOOP_RATE
    // @DisplayName: Scheduling main loop rate
    // @Description: This controls the rate of the main control loop in Hz. This should only be changed by developers. This only takes effect on restart. Values over 400 are considered highly experimental.
    // @Values: 50:50Hz,100:100Hz,200:200Hz,250:250Hz,300:300Hz,400:400Hz
    // @RebootRequired: True
    // @User: Advanced
    AP_GROUPINFO("LOOP_RATE",  1, AP_Scheduler, _loop_rate_hz, SCHEDULER_DEFAULT_LOOP_RATE),

Reply:
done